src/main/java/uk/ac/ebi/ampt2d/registry/service/mail/MailService.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.registry.service.mail;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MailService {
+
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    @Value("${mail.notify:false}")
+    private Boolean nofify;
+
+    @Value("${mail.to}")
+    private String to;
+
+    @Value("${mail.from}")
+    private String from;
+
+    @Value("${mail.subject}")
+    private String subject;
+
+    public void send(String text) {
+        if (nofify == true) {
+            SimpleMailMessage message = new SimpleMailMessage();
+            message.setTo(to);
```

Comment:
you can verify the fields before directly assigning like, not null, trim.. as necessary

Reply:
if the fields are null the application context won't load and if invalid, it will be caught in the exception handler.